src/tests/add.spec.ts

```diff
@@ -32,4 +36,38 @@ describe('MiddlewareRegistry.add', () => {
     expect(middlewareB.middleware).not.toHaveBeenCalled()
     expect(registry['registry'].size).toEqual(2)
   })
+
+  it.each([{ methods: ['GET'] }, undefined, { methods: ['GET', 'POST'] }])(
+    'should log a warning for duplicate entries when debug is false',
+    async (config) => {
+      // GIVEN a request to API endpoint A, and 2 middlewares applied to the same route
+      const request = { url: '/api/a' } as NextApiRequest
+      const registry = new MiddlewareRegistry(request)
+      const consoleWarnSpy = jest.spyOn(console, 'warn')
+
+      expect(registry['registry'].size).toEqual(0)
+
+      // WHEN endpoints A and B are added, and registry is executed
```

Comment:
I think this comment needs to be updated? Same deal on line 66.

Reply:
Yep good catch.